modules/boost.process/1.87.0/MODULE.bazel

```diff
@@ -0,0 +1,27 @@
+module(
+    name = "boost.process",
+    version = "1.87.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108300,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "boost.algorithm", version = "1.87.0")
+bazel_dep(name = "boost.asio", version = "1.87.0")
```

Comment:
`bazel_dep(name = "boost.asio", version = "1.87.0.bcr.1")` try always to use the currenlty newest one...